pkg/termcorder/termcorder_test.go

```diff
@@ -0,0 +1,116 @@
+package termcorder_test
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/haguro/termcord/pkg/termcorder"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTermcording(t *testing.T) {
+	t.Parallel()
+	cfg := &termcorder.Config{}
+	tc, err := termcorder.NewTermcording(cfg)
+	assert.IsType(t, &termcorder.Termcording{}, tc)
+	assert.NoError(t, err)
+}
+
+func TestTermcordingFromFlags(t *testing.T) {
+	t.Parallel()
+	t.Run("Test setting flags", func(t *testing.T) {
```

Comment:
I don't see much value in calling `t.Run` once—it's really designed to be used in a loop over many test cases, I think. We don't really gain anything from using it here except some extra clutter and indentation caused by the function literal. We might just as well write that code directly in the test, or—even better—make it a separate named test.